_episodes/07-docker-image-examples.md

```diff
@@ -0,0 +1,57 @@
+---
+title: "Examples of Using Container Images in Practice"
+teaching: 20
+exercises: 0
+questions:
+- "How can I use Docker for my own work?"
+objectives:
+- "Use existing container images and Docker in a research project."
+keypoints:
+- "There are many ways you might use Docker and existing container images in your research project."
+---
+
+Now that we have learned the basics of working with docker images and containers,
+let's apply what we learned to an example workflow.
+
+You may choose one or more of the following examples to practice using containers.
+
+## Jekyll Website Example
+
+In this [Jekyll Website example](../_extras/e02-jekyll-lesson-example), you can practice
+rendering this lesson website locally using Docker and a Jekyll a container image.
+This example helps avoid a complicated software install by using a container image.
```

Comment:
I'd like to make this more clear and direct: Rendering the website in a container avoids a complicated software installation; instead of installing Jekyll and all the other tools needed to create the final website, all the work can be done in the container.

Reply:
```suggestion
Rendering the website in a container avoids a complicated software installation; instead of installing Jekyll and all the other tools needed to create the final website, all the work can be done in the container.
```